src/tools/runk/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "runk"
+version = "0.0.1"
+authors = ["The Kata Containers community <kata-dev@lists.katacontainers.io>"]
+description = "runk: Kata OCI container runtime based on Kata agent"
+license = "Apache-2.0"
+edition = "2018"
+
+[dependencies]
+libcontainer = { path = "./libcontainer" }
+rustjail = { path = "../../agent/rustjail", features = ["standard-oci-runtime"] }
+oci = { path = "../../libs/oci" }
+logging = { path = "../../libs/logging" }
+liboci-cli = { git = "https://github.com/containers/youki.git" }
```

Comment:
Hi! I am a maintainer of youki. I have the plan to publish `liboci-cli` to crates.io soon.
https://github.com/containers/youki/issues/586

Reply:
Thank you for the information. It's awesome!

Reply:
Any update on publishing on crates.io @utam0k?